common/db.go

```diff
@@ -94,6 +101,13 @@ var schema = `
 	);
 
 	CREATE INDEX IF NOT EXISTS idx_winningtickets_sessionid ON winningTickets(sessionID);
+
+	CREATE TABLE IF NOT EXISTS blockheaders (
+		number int64 PRIMARY KEY,
+		parent STRING,
+		hash STRING,
+		logs TEXT
```

Comment:
Since `encodeLogsJSON()` returns a `[]byte` and `decodeLogsJSON()` accepts a `[]byte`, I think storing `logs` as a `BLOB` would allow us to skip the extra `string <-> []byte` conversion step.

Reply:
Fixed in ced063e